bonita-integration-tests/bonita-integration-tests-client/src/main/java/org/bonitasoft/engine/search/SearchActivityInstanceIT.java

```diff
@@ -1283,4 +1285,30 @@ public ProcessDefinition deployProcessWithActorAndTestConnectorLongToExecute(fin
                 TestConnectorLongToExecute.class, "TestConnectorLongToExecute.jar");
     }
 
+
+    @Test
```

Comment:
the other one was on search term, but we could have yes

i can make the change